packages/doc/src/components/CodeBlock/CodeBlock.jsx

```diff
@@ -281,7 +281,13 @@ const CodeBlock = ({ children, reactLive, center }) => {
       language="jsx"
     >
       {({ className, style, tokens, getLineProps, getTokenProps }) => (
-        <Pre className={className} style={style}>
+        <Pre
+          className={className}
+          style={{
+            ...style,
+            border: '1px solid #DBDBE8',
```

Comment:
Why is this inline?

Line 27 declare `Pre` styled-component inheriting gray colors from theme, what you think of adding this style to the component? If you can't find the same gray, use one the look alike from our pallete

Reply:
nice tip! done!